python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -230,8 +234,14 @@ def get_openlineage_facets(self, task_instance):  # skipcq: PYL-W0613
             )
         ]
 
-        output_dataset: list[OpenlineageDataset] = [OpenlineageDataset(namespace=None, name=None, facets={})]
+        output_dataset: list[OpenlineageDataset] = []
         if self.output_table is not None and self.output_table.openlineage_emit_temp_table_event():
+            self.output_table.conn_id = task_instance.xcom_pull(
```

Comment:
We shall add a comments here to remove it in future

Reply:
+1

Reply:
Added a comment